onebusaway-android/src/google/java/org/onebusaway/android/map/googlemapsv2/BaseMapFragment.java

```diff
@@ -518,7 +520,7 @@ public void setOnFocusChangeListener(OnFocusChangedListener onFocusChangedListen
     final Handler mStopChangedHandler = new Handler();
 
     public void onFocusChanged(final ObaStop stop, final HashMap<String, ObaRoute> routes,
-            final Location location) {
+                               final Location location) {
```

Comment:
From a quick look at `onFocusChanged()`, it looks like `mFocusStopId` isn't reset to `null` when focus is lost:

~~~
                if (stop != null) {
                    mFocusStopId = stop.getId();
                    //Log.d(TAG, "Focused changed to " + stop.getName());
                } else {
                    //Log.d(TAG, "Removed focus");
                }
~~~

I believe the ELSE block should have `mFocusStopId = null`, although you'd need to be sure there aren't any other unintended consequences of this.  `onSaveInstanceState()` is the only place where this value is used in `BaseMapFragment`, so it looks like we currently retain the previously selected stopId here, even if the stop isn't currently in focus.  Not sure off the top of my head if this was intended behavior.